examples/do_apply_ica_replace_mean_std.py

```diff
@@ -53,7 +53,11 @@
 
 # compare filtered and unfiltered data
 for idx in range(0, len(pca_mean_filt)):
-    print pca_mean_filt[idx], pca_mean_replaced_filt[idx], pca_mean_replaced_unfilt[idx]
+    print '%10.6f\t%10.6f\t%10.6f' % (pca_mean_filt[idx], pca_mean_replaced_filt[idx], pca_mean_replaced_unfilt[idx])
+    if idx >= 9:
+        break
 
 for idx in range(0, len(pca_pre_whitener_filt)):
     print pca_pre_whitener_filt[idx], pca_pre_whitener_replaced_filt[idx], pca_pre_whitener_replaced_unfilt[idx]
+    if idx >= 9:
+        break
```

Comment:
simply doing below would work.. 
```
print pca_pre_whitener_filt[:10]
```
also, nice padding :)

Reply:
print pca_pre_whitener_filt[:10], print pca_pre_whitener_replaced_filt[:10], print pca_pre_whitener_replaced_unfilt[:10]
If I use this, wouldn't all the entries be plotted below each other? I did it this way because I wanted the three corresponding entries in one line.

Reply:
Ah okay, then this is fine 👍